plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -853,18 +870,33 @@ static bool prepare_audio(struct ffmpeg_data *data,
 	return true;
 }
 
-static void receive_audio(void *param, struct audio_data *frame)
+/* given a bitmask for the selected tracks and the mix index this returns the stream index which will be passed to the muxer*/
+int getTrackOrder(int track_config, int mix_index)
+{
+	int position = 0;
+	int i;
```

Comment:
Any particular reason for declaring `i` before the `for` loop instead of in the loop's init?